cfgov/jinja2/v1/_includes/molecules/info-unit-2.html

```diff
@@ -0,0 +1,136 @@
+{# ==========================================================================
+
+   Info Unit
+
+   ==========================================================================
+
+   Description:
+
+   Create an info-unit molecule when given:
+
+   value:                        Object defined from a StreamField block.
+
+   value.modifier:               A modifier CSS class used to alternate
+                                 the info unit from stacked to inline layouts.
+
+   value.image:                  (Optional) An object containing:
+
+   value.image.is_decorative:    Whether the image is a bg image or not.
+                                 Default is false.
+
+   value.image.is_square:        Whether the image is square or not.
+                                 Default is false.
+
+   value.image.url:              String for the image URL.
+
+   value.image.alt:              String for image alt text.
+
+   value.heading:                The heading for the info unit.
```

Comment:
Possibly. I've thought about doing that. It would be more DRY if we had a `HeadingBlock` that rolled up the three new Heading fields in one. I'll play with that before finishing this PR.